content/foundations/getting-started.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Getting started
```

Comment:
I would have expected a "setup & install" post with this title. Could we maybe change it to something like "Introduction" or similar?

Reply:
I think a "setup & install" makes sense for the "Build" section, but not for "Interface guidelines" in this case. I'm treating the information architecture and how we break these sections apart as a different pull request, we can improve this out iteratively!

For reference @emilybrick has been thinking about IA [in this Google Doc](https://docs.google.com/document/d/1YcIp34rA8jC7df2Vm5HUCCd1k2O_EMsdI2eO-vMiSb0/edit#heading=h.b161blt5ckx8) (only available for hubbers)